amlb/datautils.py

```
@@ -143,29 +148,52 @@ def reorder_dataset(
+    n_cols = len(columns)
+    src = n_cols + 1 + target_src if target_src < 0 else target_src
+    dest = n_cols + 1 + target_dest if target_dest < 0 else target_dest
+
+    if src == dest:
+        return None
+
+    ori = list(range(n_cols))
+    if src < dest:
+        return ori[:src] + ori[src + 1 : dest] + [src] + ori[dest:]
+    else:
+        return ori[:dest] + [src] + ori[dest:src] + ori[src + 1 :]
```

Ruff (0.14.6)

Consider `[*ori[:dest], src, *ori[dest:src], *ori[src + 1:]]` instead of concatenation

Replace with `[*ori[:dest], src, *ori[dest:src], *ori[src + 1:]]`

(RUF005)